NBitcoin/Network.cs

```diff
@@ -807,6 +814,8 @@ public ChainName ChainName
 
 		public static Network TestNet => Bitcoin.Instance.Testnet;
 
+		public static Network TestNet4 => Bitcoin.Instance.Testnet4;

```

Comment:
Remove.  Just use Bitcoin.Instance.Testnet4. testnet4 is a bitcoin-only concept.

Reply:
Done